packages/components/src/internal/components/editable/utils.test.ts

```diff
@@ -762,7 +765,8 @@ describe('getValidatedEditableGridValue', () => {
             },
             value: true,
         });
-        expect(getValidatedEditableGridValue(13, dateTimeCol).message).toBe(undefined);
+        // TODO: Why is this considered a valid date?
+        // expect(getValidatedEditableGridValue(13, dateTimeCol).message).toBe(undefined);
```

Comment:
as mentioned above, this was just added since moment did consider a number a valid date. If the new code considers this invalid, I think that is even better.

Reply:
Same as above. I was able to revert any changes needed here as behavior now aligns with the test.